src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -208,33 +190,37 @@ class ElectronicDeliveryForm extends React.Component {
           <legend>Page Number Range (Max 50 pages)</legend>
           <h3>Page Number Range (Max 50 pages)</h3>
 
-          <div className={`nypl-year-field ${errorClass['starting-page']}`}>
-            <label htmlFor="starting-page" id="starting-page-label">Start Page
+          <div className={`nypl-year-field ${errorClass['start-page']}`}>
+            <label htmlFor="start-page" id="start-page-label">Start Page
+              <span className="nypl-required-field">&nbsp;(Required)</span>
             </label>
             <input
-              id="starting-page"
+              id="start-page"
               type="number"
+              required
               className="form-text"
-              aria-labelledby="starting-page-label"
-              name="starting-page"
-              value={this.state.form['starting-page']}
-              onChange={(e) => this.handleUpdate(e, 'starting-page')}
+              aria-labelledby="start-page-label"
+              name="startPage"
+              value={this.state.form['startPage']}
```

Comment:
Since there's no hyphen anymore in the variable name, you can do `this.state.form.startPage` and `this.state.form.endPage`.